PhysicsTools/NanoAOD/python/custom_jme_cff.py

```diff
@@ -681,10 +681,23 @@ def ReclusterAK4CHSJets(proc, recoJA, runOnMC):
   proc.jetSequence.insert(proc.jetSequence.index(proc.pileupJetId94X), getattr(proc, pileupJetId80X))
 
   proc.updatedJetsWithUserData.userInts.puId80XfullId = cms.InputTag('pileupJetId80X:fullId')
-  run2_jme_2016.toModify(proc.updatedJetsWithUserData.userFloats, puId80XDisc = cms.InputTag("pileupJetId80X:fullDiscriminant"))
+  proc.updatedJetsWithUserData.userFloats.puId80XDisc = cms.InputTag("pileupJetId80X:fullDiscriminant")
 
-  proc.jetTable.variables.puId = Var("userInt('puId80XfullId')", int, doc="Pilup ID flags with 80X (2016) training")
   run2_jme_2016.toModify(proc.jetTable.variables, puIdDisc = Var("userFloat('puId80XDisc')",float,doc="Pilup ID discriminant with 80X (2016) training",precision=10))
+  run2_jme_2016.toModify(proc.jetTable.variables, puId = Var("userInt('puId80XfullId')",int,doc="Pileup ID flags with 80X (2016) training"))
```

Comment:
what do you intent to do in the jmenano ?

the full set of puget Id as stored in central nano are more complex depending on the nano version 
https://github.com/cms-sw/cmssw/blob/master/PhysicsTools/NanoAOD/python/jets_cff.py#L268-L278

Reply:
For the reclustered AK4 CHS jets in JMEnano, we would like the Pileup Id discriminant and flags to be set according to the respective training campaigns. The settings here from the main (https://github.com/cms-sw/cmssw/blob/master/PhysicsTools/NanoAOD/python/jets_cff.py#L268-L278) does not need to be repeated again in JMEnano. This line is needed at the moment because the UL2016 training is still not included in the main nano.